src/styles/theme/oc-sidebar.scss

```diff
@@ -0,0 +1,112 @@
+.oc-sidebar {
+  display: flex;
+  flex-flow: column wrap;
+  width: $width-medium;
+  height: 100%;
```

Comment:
we always want the sidebar to cover 100% of the viewport height, correct? In that case we should set it to `100vh`, just to be sure.

Reply:
Yeah, usually I try to stay away from `vh` since on some devices it doesn't behave that nice with e. g. toolbar. But you're right, it makes more sense here. 👍